insights/combiners/sys_vmbus_devices.py

```diff
@@ -0,0 +1,38 @@
+"""
+VMBus device info
+=================
+"""
+from insights.core.plugins import combiner
+from insights.parsers.sys_vmbus import SysVmbusDeviceID, SysVmbusClassID
+
+
+@combiner(SysVmbusDeviceID, SysVmbusClassID)
+class SysVmBusDeviceInfo(object):
+    '''
+    Combiner to access all the VMBus devices.
+
+    Attributes:
+        devices (list): The list is dict.
```

Comment:
@psachin would you mind to add the dictionary keys to the documentation here?  It should be clear in the docs how to use this component without having to look at the code.  The examples do show all of the keys, but a dev wouldn't know that without investigating the code.  Thanks.